pkg/pillar/cmd/downloader/resolveconfig.go

```diff
@@ -83,9 +83,13 @@ func maybeRetryResolve(ctx *downloaderContext, status *types.ResolveStatus,
 	// Increment count; we defer clearing error until success
 	// to avoid confusing the user.
 	status.RetryCount++
-	errStr := fmt.Sprintf("Retry attempt %d after %s",
-		status.RetryCount, status.OrigError)
-	status.SetErrorNow(errStr)
+	severity := types.GetErrorSeverity(status.RetryCount, time.Duration(status.RetryCount)*retryTime)
+	errDescription := types.ErrorDescription{
+		Error:               status.OrigError,
+		ErrorRetryCondition: fmt.Sprintf("Retrying; attempt %d", status.RetryCount),
```

Comment:
Doesn't this also have a timer hence should say Will retry in %s; as the others?

Reply:
Actually this is a place where we start new iteration of try. So, in this case we still want to keep an old error and let next attempt to change/clean it.

Reply:
Ah - I forgot that part. You're all good then.